DockerfileLint

```diff
@@ -0,0 +1,11 @@
+FROM python:3.6
+
+WORKDIR /app
+
+ENV PIP_CACHE ""
+ENV SCRIPT ""
+COPY ./Makefile /app
```

Comment:
aren't all these files already available through the run-time mount?

Reply:
for linting I only mount the `.cache\pip`.

Reply:
Any specific reason for that? We still need access to the source code for linting, and mounting the folder seems easier than building a new image every time